src/connections/destinations/catalog/actions-fullstory/index.md

```diff
@@ -13,19 +13,19 @@ versions:
 
 [FullStory](https://www.fullstory.com/){:target="_blank"} lets product and support teams easily understand everything about the customer experience. The Segment integration for FullStory helps accurately identify your customers within the FullStory dashboard.
```

Comment:
Not sure if it's an artifact of how GH displays it, but the target window parameter displays in plaintext in rich diff view. 
![Image 2022-07-25 at 10 08 39 AM](https://user-images.githubusercontent.com/72521016/180797275-9844652a-1a9c-4921-bc9d-073614c606ad.jpg)

Reply:
Also, this appears to be the only link with that target param. Perhaps we just remove it?

Reply:
Do we want the FullStory link to open in a new tab? If so, we could add the same target to the FullStory link in the FullStory Cloud Mode (Actions) doc. Otherwise, we can definitely remove the target here to have this link open in the same tab.

Did Segment author the original docs for us? Wondering if this is a pattern they set that might be worth following.

Reply:
Yes, they did author the original docs. It would be a better experience to have all links open in a new window.

Reply:
SGTM. I added `target="_blank"` to all external FullStory links, but left the internal Segment links as is. Let me know if you think we should go a different direction.